core-services/importExport/src/main/java/org/silverpeas/core/importexport/versioning/VersioningImport.java

```diff
@@ -130,7 +113,7 @@ public int importDocuments(String objectId, String componentId, List<AttachmentD
         boolean removed = FileUtils.deleteQuietly(attachmentImportExport.getAttachmentFile(
             attachment));
         if (!removed) {
-          SilverTrace.error("versioning", "VersioningImportExport.importDocuments()",
+          SilverTrace.error("versioning", "VersioningImport.importDocuments()",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "versioning" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "VersioningImport.importDocuments()" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1192)